### PR TITLE
Removes the last of the color macros

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Removes the last of the color macros.

Also there was an issue with being able to utilize the space movement of superfart if you had both toxic farts and superfarts. That's fixed.

:cl: Fox McCloud
fix: Fixes not being able to propel yourself through space by farting if you possessed both superfart and toxic farts
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -826,34 +826,34 @@
...
 
 			// Process toxic farts first.
 			if(TOXIC_FARTS in mutations)
-				for(var/mob/M in range(location,aoe_range))
-					if(M.internal != null && M.wear_mask && (M.wear_mask.flags & AIRTIGHT))
+				for(var/mob/living/carbon/C in range(location, 2))
+					if(C.internal != null && C.wear_mask && (C.wear_mask.flags & AIRTIGHT))
 						continue
-					// Now, we don't have this:
-					//new /obj/effects/fart_cloud(T,L)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/emote.dm]: On Goon, this is just a visual yellow cloud that adds jenkem to you; we just do that, below, invisibly, without the cloud (and on Goon, the cloud has no actual icon, so it's technically invisible).

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -826,34 +826,34 @@
...
-				message = "<b>[src]</b> [pick("passes wind","farts")]."
+				message = "<b>[src]</b> [pick("passes wind", "farts")]."
 			m_type = 2
 
 			var/turf/location = get_turf(src)
-			var/aoe_range=2 // Default
 
 			// Process toxic farts first.
 			if(TOXIC_FARTS in mutations)
-				for(var/mob/M in range(location,aoe_range))
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/emote.dm]: Poisoning ghosts....with ***farts***

File: code/modules/mob/living/carbon/human/emote.dm (hunk 1 above)
Comment (taukausanake) [reply code/modules/mob/living/carbon/human/emote.dm]: Having a cloud would be hilarious, but I think I can see why we wouldn't have one

**Comment by taukausanake** [comment]
Why wouldn't you be able to propel yourself with farts in space? It's thrust, you should be able to, unless clothes and/or spacesuits block it

File: code/modules/mob/living/carbon/human/emote.dm [hunk 3]
```diff
@@ -826,34 +826,34 @@
 				return
 //			playsound(loc, 'sound/effects/fart.ogg', 50, 1, -3) //Admins still vote no to fun
 			if(locate(/obj/item/storage/bible) in get_turf(src))
-				to_chat(viewers(src), "<span class='warning'><b>[src] farts on the Bible!</b></span>")
-				var/image/cross = image('icons/obj/storage.dmi',"bible")
-				var/adminbfmessage = "\blue [bicon(cross)] <b><font color=red>Bible Fart: </font>[key_name(src, 1)] (<A HREF='?_src_=holder;adminmoreinfo=\ref[src]'>?</A>) (<A HREF='?_src_=holder;adminplayeropts=\ref[src]'>PP</A>) (<A HREF='?_src_=vars;Vars=[UID()]'>VV</A>) (<A HREF='?_src_=holder;subtlemessage=\ref[src]'>SM</A>) ([admin_jump_link(src)]) (<A HREF='?_src_=holder;secretsadmin=check_antagonist'>CA</A>) (<A HREF='?_src_=holder;Smite=[UID()]'>SMITE</A>):</b>"
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/emote.dm]: Why did this even have a color macro to begin with. The `\blue` ends up doing nothing.

File: code/modules/mob/living/carbon/human/emote.dm (hunk 2 above)
Comment (KasparoVy) [reply code/modules/mob/living/carbon/human/emote.dm]: outspooked

**Comment by theColdflame** [comment]
fart jokes are funny

**Pull request merged**